### DISCUSSION TITLE
Can't add Microsoft.Windows.SDK.Contracts package to the project using “Cake”

### DISCUSSION THREAD

**Discussion by Bogdan029** [cake-build/cake#3285] [Q&A] [answered]
I want to build my solution using cake, and I have an issue. I am not able to add Microsoft.Windows.SDK.Contracts Nuget package to the project. I use this command :
`#addin nuget:?package=Microsoft.Windows.SDK.Contracts&version=10.0.19041.1&loaddependencies=true`
and I get errors 
`C:\sen_cake\ble-module\Module.BLE\Plugin.BLE\Core\BluetoothLE.cs(11,15): error CS0234: The type or namespace name 'Devices' does not exist in the namespace 'Windows' (are you missing an assembly reference?) [C:\sen_cake\ble-module\Module.BLE\Plugin.BLE\Plugin.BLE.csproj]`
All my build.cake file :

```
#addin nuget:?package=Microsoft.Windows.SDK.Contracts&version=10.0.19041.1&loaddependencies=true

var target = Argument("target", "Build");
var configuration = Argument("configuration", "Release");

//////////////////////////////////////////////////////////////////////
// TASKS
/////////////////////////////////////////////////////////////////////
    
Task("Build")

    .Does(() =>
{

    DotNetCoreBuild("../Module.BLE/Module.BLE.sln", new DotNetCoreBuildSettings
    {
        Configuration = configuration,
    });
});


//////////////////////////////////////////////////////////////////////
// EXECUTION
//////////////////////////////////////////////////////////////////////

RunTarget(target);
```

**Comment by augustoproiete** [comment]
@Bogdan029 Could you provide a sample project that reproduces the error above?

**Reply by Bogdan029** [reply]
@augustoproiete Its not my project, but it contains part of my project, with the same errors.
[ClassLibrary1.zip](https://github.com/cake-build/cake/files/6149245/ClassLibrary1.zip)

**Comment by nils-a** [comment, marked as answer]
@Bogdan029 That error comes from your code, not from Cake. Most probably you should be trying to add `Microsoft.Windows.SDK.Contracts` to your projects, not to Cake.

Using the solution you provided above, running `dotnet build` (no Cake involved) yields an identical error.
Running 

```
dotnet add .\ClassLibrary1\ClassLibrary1.csproj package Microsoft.Windows.SDK.Contracts
dotnet build 
```
works fine.

After that, the Cake build also works fine.

**Reply by Bogdan029** [reply]
Thanks
